ClosedXML.Report/Utils/ConvertExtensions.cs

```diff
@@ -50,15 +50,12 @@ public static bool AsBool(this string value)
                 return false;
             switch (value.ToLower())
             {
-                case "0":
-                case "0.0":
-                case "0,0":
-                case "-":
```

Comment:
Why was this necessary? Zeros are usually treated as "falsy".

~It bothers me that we are making a breaking change here.~ Nah, I see, the methods was not used anywhere 🤦

Reply:
We left only the values that seem logical to us for this parameter, but I agree to bring it back